packages/scans/README.md

```diff
@@ -2,3 +2,7 @@
 
 The following repository contains common code and helper functionalities to run the ZAP baseline scan and full scan with
 GitHub Actions.
+
+## Contributing
+
+See [CONTRIBUTING.md](./CONTRIBUTING.md).
```

Comment:
I'd say this is redundant, this is a well known file.

Reply:
I would argue that it doesn't hurt, either, and makes the repo more friendly to first-time contributors. Also, this shows people that they should actually read the `CONTRIBUTING.md` - there are plenty of repos where it isn't really maintained.

Reply:
GitHub already shows the file when contributing (which more often than not is just ignored).

Reply:
I don't have super strong feelings here. If you feel strongly about this, I would remove it. Though I still don't see the harm of including it, while I see a bit of value.

Reply:
Leave as is.